src/__experimental__/components/textbox/__snapshots__/textbox.spec.js.snap

```diff
@@ -1,61 +1,209 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Textbox renders with InputPresentation and Input and all props passed to Input 1`] = `
-<FormField
-  hasError={false}
-  hasInfo={false}
-  hasWarning={false}
-  id="mocked-guid"
-  info={Array []}
-  inputWidth={70}
-  labelWidth={30}
-  name="mocked-guid"
-  onBlur={[Function]}
-  onChange={[Function]}
-  size="medium"
-  styleOverride={Object {}}
-  useValidationIcon={false}
-  validations={Array []}
-  warnings={Array []}
+<styled.div
```

Comment:
We need to remove the theme object from this snapshot please as any time a new property is added this snapshot will need to be updated. There is the `noThemeSnapshot` helper function `__spec_helper__/enzyme-snapshot-helper`